assemblies/assembly-rhdh-integration-aks.adoc

```diff
@@ -9,12 +10,6 @@ This integration requires the deployment of {product-short} on {aks-short} using
 * The Helm chart
 * The {product} Operator
 
-include::modules/admin/proc-rhdh-deploy-aks.adoc[leveloffset=+1]
```

Comment:
Is this PR also restructuring the integration content?

Reply:
This PR separates the integration content from the installation/deployment content > relocates the installation content to the new installation guide > leaves the integration content in the Admin guide. No structural or content changes are made directly to the sections that are specific to integration.